doc/src/manual/performance-tips.md

```diff
@@ -609,22 +555,14 @@ to perform a core computation. Where possible, it is a good idea to put these co
 in separate functions. For example, the following contrived function returns an array of a randomly-chosen
 type:
 
-```@meta
-DocTestSetup = quote
-    import Random
-    Random.srand(1234)
-end
-```
-
-```jldoctest
+```jldoctest; setup = :(using Random; srand(1234))
 julia> function strange_twos(n)
            a = Vector{rand(Bool) ? Int64 : Float64}(undef, n)
            for i = 1:n
                a[i] = 2
            end
            return a
-       end
-strange_twos (generic function with 1 method)
+       end;
```

Comment:
Maybe that's just me, but I find it weird to end functions with `;` just to avoid printing it.

Reply:
Maybe, but I rather save the vertical space.